arches/app/media/css/abstracts/_breakpoint-settings.scss

```diff
@@ -4,7 +4,9 @@
 $breakpoints: (
     mobile: (min-width: 320px),
     tablet: (min-width: 740px),
+    tablet-max: (max-width: 740px),
```

Comment:
Yes, all the previous work was done with max-width's instead of min-width's and made sense at the time. When the initial sass structure was introduced there was only min-width's. So rather an changing what was there I thought we could just add some "max" breakpoints and use them, max just refer to using max-width. If you need me to change anything let me know, no problem.